documentation/modules/auxiliary/admin/networking/cisco_dcnm_auth_bypass.md

```diff
@@ -0,0 +1,36 @@
+## Description
+
+This module exploits CVE-2019-15975 which affects Cisco DCNM 11.2.1 at least. Thanks to this exploit you can add an admin account with any credentials you want. Then you can login to the web interface of Cisco DCNM with those credentials. The only necessary condition is the more or less recent connection of an admin as this exploit uses a kind of session stealing. 
```

Comment:
> is the more or less recent connection

Do you have any supporting details around this? Are we talking 30 seconds, 15 minutes, anything that would be more descriptive?

Reply:
I have no measure about that but it is rather a matter of hours.

Reply:
Adding that in could be helpful. Being in the realm of hours definitely makes this applicable in more scenarios.